src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import styles from './Pagination.module.css';
+import { IoMdArrowDropleft } from 'react-icons/io';
+import { IoMdArrowDropright } from 'react-icons/io';
+
+function Pagination({ postPerPage, totalPosts }) {
+    const pageNumbers = [];
+
+    for (let i = 1; i <= Math.ceil(totalPosts / postPerPage); i++) {
+        pageNumbers.push(i);
+    }
```

Comment:
this can be more expressive with something like:

```suggestion
    const totalPages = Math.ceil(totalPosts / postPerPage);
    const pageNumbers = Array.from({ length: totalPages }, (_, i) => i + 1);
```